pyproject.toml

```diff
@@ -1,3 +1,154 @@
+[build-system]
+requires = ["setuptools >= 61.1.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "optuna"
+description = "A hyperparameter optimization framework"
+authors = [
+  {name = "Takuya Akiba"},
+  {email = "akiba@preferred.jp"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
In `setup.py`, "Programming Language :: Python :: 3.11" is also listed.

https://github.com/optuna/optuna/blob/master/setup.py#L175